iptables/iptables.go

```diff
@@ -423,10 +424,18 @@ func existsRaw(table Table, chain string, rule ...string) bool {
 	return strings.Contains(string(existingRules), ruleString)
 }
 
-func filterOutput(output []byte, args ...string) []byte {
-	// ignore iptables' message about xtables lock
-	if strings.Contains(string(output), xLockWaitMsg) {
+// Maximum duration that an iptables operation can take
+// before flagging a warning.
+const opWarnTimeMsec = 2000
+
+func filterOutput(start time.Time, output []byte, args ...string) []byte {
+	// Flag operations that have taken a long time to complete
+	if time.Now().Sub(start) > opWarnTimeMsec*time.Millisecond {
```

Comment:
you can use time.Since() 
```
// Since returns the time elapsed since t.
// It is shorthand for 
time.Now().Sub(t).
func Since(t Time) Duration {
	return Now().Sub(t)
}
```

Reply:
Ah, that's useful.  Ok.